apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -113,6 +113,11 @@ typedef enum {
 #include "wm_test_table.h"
 extern void wm_run_stress_test(void *arg);
 extern void wm_test_on_off(void *arg);
+extern void wm_test_interop(void *arg);
+extern void wm_interop_add_ap(void *arg);
+extern void wm_interop_display_ap(void *arg);
+wifi_manager_ap_auth_type_e wm_get_auth_type(const char *method);
```

Comment:
you don't need to add function list if you added It in there.
WT_MEMBER_POOL(WT_TEST_INTEROP, _wt_interop_test, _wt_parse_interop, "interop")